### ISSUE TITLE
Exclude suspended plugins in sitemap

### ISSUE THREAD

**Issue by halkeye** [jenkins-infra/plugin-site#1203]
Problem with the [ci-skip](https://plugins.jenkins.io/ci-skip) page

Even though this plugin is no longer distributed it shows up in google results. We should tell google to not index this.

**Comment by KitsuneKode** [comment]
@krisstern this is fixed, right? Robot.txt is already in the project

**Comment by zbynek** [comment]
`robots.txt` is [the wrong way of removing stuff from google](https://support.google.com/webmasters/thread/186122187?hl=en&msgid=186123307) -- title updated.

The issue is not fixed, feel free to work on it. 

Btw. please don't ping people by name unless they're already involved in an issue.

**Comment by KitsuneKode** [comment]
@zbynek sorry about that, I will keep that in mind from now on. 

I will research on this topic, and cross-check here.

**Comment by halkeye** [comment]
ah, noindex does seem like the better idea, and easier to implement (its just helmet right?)

+1 z

**Comment by KitsuneKode** [comment]
> ah, noindex does seem like the better idea, and easier to implement (its just helmet right?)
> 
> +1 z

Yes, I also found that adding noindex will prevent the indexing. Should I go with that approach as well?

**Comment by zbynek** [comment]
@halkeye do you think using `noindex` without removing them from the sitemap is sufficient to close this issue?

**Comment by halkeye** [comment]
sounds like it might. Its been a long time since i did any of the seo related stuff.
